content/docs/learning-guide/tutorials/entities-and-prefabs/override-a-prefab.md

```diff
@@ -96,7 +111,92 @@ Once an override has been registered, it will exist until explicitly removed. To
+            }
+        }
+    }
+}
+```
+
+{{< note >}}
+You can create a new settings registry file or reuse existing settings registry files that you have already created.
+{{< /note >}}
+
+### Override visualization
+
+When you make an override edit in the inspector, you will notice that a blue circle appears before the property label and the label text is bolded. In the component header, a blue circle is also shown near the text, which indicates that an owning property value has been overridden.
+
+{{< image-width src="/images/learning-guide/tutorials/entities-and-prefabs/prefab-override-component-property.png" width="450" alt="Overriding a component property." >}}
+
+If a component is added as an override, a different blue circle with a plus sign appears next to the component name instead.
+
+{{< image-width src="/images/learning-guide/tutorials/entities-and-prefabs/prefab-override-add-component-override.png" width="450" alt="Adding a component as an override." >}}
+
+### Revert property overrides
+
+With the inspector override support, you can now revert overridden property values respectively. Unlike reverting overrides on an entity, you can selectively revert the property you want:
+
+1. In Entity Inspector, **right-click** on the property label or the blue circle and choose **Revert override** from the context menu.
+1. Notice that the property no longer has any indication of having an override.
+
+{{< image-width src="/images/learning-guide/tutorials/entities-and-prefabs/prefab-override-revert-component-property-override.png" width="450" alt="Reverting a component property override." >}}
+
+{{< note >}}
+Once a property value is overridden to non-default value, the override edit can only be removed by doing a revert. In other words, setting the value back to default does not remove the override edit.
+{{< /note >}}
+
+In addition, you are able to revert all property overrides that are owned by a component:
+
+1. In Entity Inspector, **right-click** on the blue circle in the component header and choose **Revert Overrides** from the context menu.
+1. Notice that all the properties no longer have any indication of having an override.
+
+{{< image-width src="/images/learning-guide/tutorials/entities-and-prefabs/prefab-override-revert-component-override.png" width="450" alt="Reverting a component added as an override." >}}
+
+For a component added as an override, you can revert the addition in the same way mentioned above. Alternatively, you can delete the component with either of the following methods:
+
+1. In Entity Inspector, **right-click** a component and choose **Delete component** from the context menu.
+1. With a component selected, press **DEL**.
+
+For a component deleted as an override, you can revert the deletion by an edit undo. Alternatively, you can achieve that by reverting all overrides on the selected entity, but be aware that it would also remove other overrides if they are present.
```

Comment:
Not sure what 'revert the deletion by an edit undo' means here. Do you mean that a user can simply 'undo' the action? If so, I don't think it's worth mentioning. Any editor operation that modifies the scene data should be undoable by default. So we don't need to mention that as a way to revert for several operations. I recommend  you delete this here and in other places where you say an action can be reverted using 'undo'

Reply:
Yes. Sure, I'll remove this.